<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>231</width>
      <height>376</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Aikaa kulunut:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Sekunnit:    </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd_number_sekunit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Minuutit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd_number_minuutit"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pisteet:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcd_number_pisteet"/>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hedelmien koordinaatit:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLineEdit" name="line_edit_ensimmainen"/>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="line_edit_toinen"/>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QTextBrowser" name="text_browser_ohje"/>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="vaihda_hedelmat">
       <property name="text">
        <string>Vaihda</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QPushButton" name="push_button_close">
       <property name="text">
        <string>Lopeta</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QCheckBox" name="check_box_ei_viivetta">
       <property name="text">
        <string>Ei viivetta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>push_button_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>747</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
